estimation/estimate_wct.py

```diff
@@ -270,7 +416,7 @@ def estimate(
     data = scaler.transform(input_data)
     wc = model.predict(data)
 
-    return wc
+    return wc.reshape(-1)
```

Comment:
Just a question - what is the original shape of wc supposed to be? So previously you needed [0][0] to get core_hours, and reshape[-1] removed that requirement?

Reply:
So the [0][0] is from the functions that only return a single estimation and I think when I merged master back into this branch (which contained some other changes to the single estimation functions) the [0][0] ended up on the functions that do bulk estimation, which doesn't work. 
I did some quick debugging to see what the keras model actually returns, which is array of shape (-1, 1), so I just reshape it to be 1-D and everything works nicely from there.